src/test/java/net/smartcosmos/dao/metadata/impl/MetadataPersistenceServiceTest.java

```diff
@@ -107,569 +77,416 @@ public void tearDown() throws Exception {
         metadataRepository.deleteAll();
     }
 
-    // region Upsert
+    // region Create
 
     @Test
     public void testCreate() throws Exception {
 
-        final String key = "key";
-        final String dataType = "BooleanType";
-        final String rawValue = "true";
-        final String entityReferenceType = "Object";
-        final String referenceUrn = "urn:uuid:" + UuidUtil.getNewUuidAsString();
-
-        MetadataUpsert create = MetadataUpsert.builder()
-            .referenceUrn(referenceUrn)
-            .entityReferenceType(entityReferenceType)
-            .rawValue(rawValue)
-            .dataType(dataType)
-            .key(key)
+        final String ownerType = "Thing";
```

Comment:
`ownerType` will never be "Thing". The type comes from `rest/thing/{type}`, so it's some arbitrary String.